### PR TITLE
fix: add amity-ak97

### LINKED ISSUE #1177: Amit Kumar Yadav
## Profile Screenshot

307ECCED-9B29-4B98-BCE8-2939137108E5-1

### PR DESCRIPTION
**Pull request by amity-ak97**
resolves #1177

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/amity-ak97.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/54621148/173427837-88c62bfa-34c4-4228-9103-5cc201514d21.png",
```
Comment (Panquesito7) [inline profiles/amity-ak97.json:2]: I recommend more of an image like this. What do you think?

image

```suggestion
    "image": "https://user-images.githubusercontent.com/51391473/173429345-e8914595-0281-4f1b-bded-afea0f6db842.png",
```

File: profiles/amity-ak97.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/51391473/173429345-e8914595-0281-4f1b-bded-afea0f6db842.png",
+    "issueId": 1177,
+    "name": "Amit Kumar Yadav",
+    "username": "amity-ak97"
```
Comment (Panquesito7) [inline profiles/amity-ak97.json:5]: It looks like there's a 4-space indentation.
```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/173429345-e8914595-0281-4f1b-bded-afea0f6db842.png",
  "issueId": 1177,
  "name": "Amit Kumar Yadav",
  "username": "amity-ak97"
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/amity-ak97.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/51391473/173429345-e8914595-0281-4f1b-bded-afea0f6db842.png",
```
Comment (Panquesito7) [inline profiles/amity-ak97.json:2]: I recommend more of an image like this. What do you think? 🙂

image

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/183324033-a83244ff-f987-4bf1-a792-9965b6c20a2d.png",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Review by AvidCoder101 — APPROVED** [review]
LGTM

**Comment by AvidCoder101** [comment]
Sorry about that, reviewed on the wrong PR.

**Pull request merged**